java/ql/lib/ext/java.lang.model.yml

```diff
@@ -37,13 +37,19 @@ extensions:
       - ["java.lang", "CharSequence", True, "charAt", "", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
       - ["java.lang", "CharSequence", True, "subSequence", "", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
       - ["java.lang", "CharSequence", True, "toString", "", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
+      - ["java.lang", "IllegalArgumentException", False, "IllegalArgumentException", "(String)", "", "Argument[0]", "Argument[-1].SyntheticField[java.lang.Throwable.message]", "value", "manual"]
```

Comment:
It would probably be nice to include a few more typical exception constructor models. Just including the subset that falls within the current top100 ends up being a bit arbitrary. There are IIRC both `(String)` and `(String,Throwable)` variations on most (all?) of them, and we very likely also want to include `Exception` and `IOException`.

Reply:
Can I include these on a separate PR since this PR was supposed to be for the top-100?

Reply:
sure